examples/plotting/wcsaxes_plotting_example.py

```diff
@@ -0,0 +1,64 @@
+"""
+=====================================
+Plotting points on a Map with WCSAxes
+=====================================
+
+This example demonstrates the plotting of point, line and arch in pixel coordinate,
+world coordinate, and SkyCoords respectively when plotting a map with WCSAxes.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.data.sample
+import sunpy.map
+from sunpy.coordinates.utils import GreatArc
+
+###############################################################################
```

Comment:
The problem is that after each block any created figures are shown and then destroyed, so the figure you create here is not available in the next block. I think the easiest way around this is to merge the two block comments into one big one here, and just have one (instead of two) blocks of code.

Reply:
David! which block of code you are referring to, can you please elaborate bit further. Above are just the import statements.

Reply:
Sorry for being unclear. I mean that all of the plotting code (including the code below this comment) needs to be not interrupted by a "comment block", which means a block starting with
```python
###############################################################################
```

Reply:
I try putting two separate comments under one "comment block". I hope this will work and comments are intelligible. I thought of putting them in steps but this is better? Please have a look David

Reply:
I tried putting two separate comments under one "comment block". I hope this will work and comments are intelligible. I thought of putting them in steps but this is better? Please have a look David